addons/binding/org.openhab.binding.mcp23017/src/main/java/org/openhab/binding/mcp23017/internal/PinMapper.java

```diff
@@ -0,0 +1,39 @@
+package org.openhab.binding.mcp23017.internal;
+
+import static com.pi4j.gpio.extension.mcp.MCP23S17Pin.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.pi4j.io.gpio.Pin;
+
+/**
+ * Created by aogorek on 16.07.17.
```

Comment:
Please follow the default openHAB format, description of class with an author tag with your fullname